saleor/graphql/utils/filters.py

```diff
@@ -70,7 +70,7 @@ def inner(qs, _, value):
     return inner
 
 
-def filter_where_range_field(qs, field, value):
+def filter_where_range_field_with_conditions(qs, field, value):
```

Comment:
Some of these functions seem similar and could use a docstring, e.g., what's the difference between this one vs `filter_where_by_range_field` or `filter_range_field` - if you're not familiar with this code, it takes a while to figure out.

Reply:
I'll update in another PR